Documentation/learning/why.md

```diff
@@ -1,5 +1,7 @@
 # Why etcd
 
+The term "etcd" originated from two ideas, unix "/etc" folder and "d"istibuted system. The "/etc" folder is a place to store configuration data for a single system whereas etcd stores configuration information for large scale distributed system. So a "d"istributed "/etc" store is "etcd", "/etc" + "d"istributed. 
```

Comment:
s/term/name
s/unix/the unix/
s/distributed system/distributed systems/

> So a "d"istributed "/etc" store is "etcd", "/etc" + "d"istributed.

Hence, a "d"istributed "/etc" is "etcd".